### PR TITLE
Guide next vs 6

### PR DESCRIPTION
**Pull request by raveling**
1. Turn’s [Jed's Github comment](https://github.com/keystonejs/keystone/issues/4962#issuecomment-808836976) into a guide.
2. Add link to guide in sidebar

A few resources on the Keystone 5 site will point to this guide soon. PR for that pending

### CODE REVIEW

File: .changeset/clever-eagles-try.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-next/website': patch
+---
+
+Add Keystone 5 vs Next guide to site
```
Comment (timleslie) [inline .changeset/clever-eagles-try.md:5]:
```suggestion
Added a Keystone 5 vs Next guide to site.
```

The comments we leave in changesets are used to generate our `CHANGELOG.md` contents. For consistency we want these changelogs to be written in **past tense** and **complete sentences** (e.g. capital letters, full stops, etc). This is somewhat motivated by pedantry on my part, but I think it makes for a better reading experience when people go to find out what has changed.

File: docs-next/components/Navigation.tsx [hunk 2]
```diff
@@ -64,6 +64,7 @@ export const Navigation = () => {
         <NavItem href="/guides/testing" isPlaceholder>
           Testing
         </NavItem>
+        <NavItem href="/guides/keystone-5-vs-keystone-next">Keystone 5 vs Next</NavItem>
```
Comment (timleslie) [inline docs-next/components/Navigation.tsx:67]: Can we move this to the top of the guides block? I think this is probably the most important guide that we have at the moment (or possibly number 2, after the command line).

File: docs-next/pages/guides/keystone-5-vs-keystone-next.mdx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+- New field types. Including a new [Document field](../guides/document-fields) field which replaces the old Content field.
+- New [configuration syntax](../apis/config) and [CLI](../guides/cli).
+- New express-independent session system.
+- New internal APIs for querying lists.
+- Completely new Admin UI.
+
+### DB adapters out. Prisma in.
+
+We’re also moving away from maintaining several database adapters to just using **Prisma**, which simplifies things. Until Prisma support MongoDB (which is planned) we’ll have a gap in our database support.
```
Comment (timleslie) [inline docs-next/pages/guides/keystone-5-vs-keystone-next.mdx:34]:
```suggestion
We’re also moving away from maintaining several database adapters to just using **Prisma**, which simplifies things. Until Prisma support MongoDB (which is [planned](https://www.notion.so/Prisma-Roadmap-50766227b779464ab98899accb98295f)) we’ll have a gap in our database support.
```

File: docs-next/pages/guides/keystone-5-vs-keystone-next.mdx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+There are also differences between Knex and Prisma for Postgres users, which mainly matter for advanced use cases like migrations and raw sql, etc.
```
Comment (timleslie) [inline docs-next/pages/guides/keystone-5-vs-keystone-next.mdx:36]:
```suggestion
There are also differences between Knex and Prisma for Postgres users, which mainly matter for advanced use cases like migrations and raw SQL, etc.
```

File: docs-next/pages/guides/keystone-5-vs-keystone-next.mdx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+## Keystone Next is production ready
+
+Thinkmill is already running Keystone Next in production (including [rugby.com.au](https://www.rugby.com.au/)) so we do rate it as stable from a runtime perspective; but from an API perspective it is still evolving fairly regularly.
+
+
+## Feature parity
+
+We’ve stripped back quite a few field types (like `File` and `HTML`) and some other functionality during the transition from 5 to 6. These will all be coming back, but **we’re not at feature parity yet**.
```
Comment (timleslie) [inline docs-next/pages/guides/keystone-5-vs-keystone-next.mdx:45]:
```suggestion
We’ve stripped back quite a few field types (like `File` and `Url`) and some other functionality during the transition from 5 to 6. These will all be coming back, but **we’re not at feature parity yet**.
```

File: docs-next/pages/guides/keystone-5-vs-keystone-next.mdx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+While parity isn’t far off, if you need something urgently that Next doesn’t include then that might answer the question for you.
+
+## Next APIs & supporting Docs are evolving
+
+Next APIs aren’s 100% stable yet. We’ve published docs for most of the APIs in Keystone Next, but there’s a lot more we have to write.
```
Comment (timleslie) [inline docs-next/pages/guides/keystone-5-vs-keystone-next.mdx:51]:
```suggestion
Next APIs aren’t 100% stable yet. We’ve published docs for most of the APIs in Keystone Next, but there’s a lot more we have to write.
```

**Review by timleslie — CHANGES_REQUESTED** [review]
Just a few typos and the such.

File: docs-next/pages/guides/keystone-5-vs-keystone-next.mdx (hunk 3 above)
Comment (raveling) [reply docs-next/pages/guides/keystone-5-vs-keystone-next.mdx:34]: this is great. i went looking for something like this link but found an old GH issue with lots of people complaining. wasn't the best optics.

**Comment by raveling** [comment]
@timleslie believe al your changes requests now fulfilled

**Review by timleslie — APPROVED** [review]
Looks good 👍

**Pull request merged**